tests/core/models/test_configs/test_painn.yml

```diff
@@ -1,17 +1,35 @@
+
 trainer: forces
 
-task:
```

Comment:
btw i think the other configs like test_equiformers_v2 also has the "task" field and can be removed